include/torch-mlir/Dialect/Torch/IR/GeneratedTorchOps.td

```diff
@@ -12089,3 +12089,27 @@ def Torch_QuantizedLinearOp : Torch_Op<"quantized.linear", [
   }];
 }
 
+def Torch_AtenTileOp : Torch_Op<"aten.tile", [
```

Comment:
Do you just need the ODS? If so, no e2e test is needed.

Reply:
yes , I've just add the ODS here.

Reply:
Then there's no need to add all the other changes, just the changes in `GeneratedTorchOps.td`

Reply:
Please see if this commit is Ok.